assets/javascripts/lib/discourse-markdown/bbcode-plugin.js

```diff
@@ -52,6 +52,9 @@ export function setup(helper) {
 
   helper.allowList([
     "div.mermaid",
+    "div.bbcode-content-center",
+    "div.bbcode-content-left",
+    "div.bbcode-content-right",
```

Comment:
@Alteras1  Old style naming feels a little inconsistent. Do you have a preference between camel  case and hyphenated? Let's pick one and stick with it unless you have other thoughts.

Reply:
I'm used to using hyphens for class names. I think the current class names you have work.

I'm also wondering if we should shorten the prefix to `bb-` or maybe use data attributes + class to keep the class names clean?

In this specific instance, I think you can probably just inline the style?

Reply:
I am fine with hyphens. I think something like "bb-right" would be fine. It'll note that it's a bbcode specific style and still keep things from being overly verbose.  I will make a note to go back later and update highlight (line 58) to match convention if you're fine with that "bb-highlight'. 

For inline vs style sheets, I was for now just following the convention we had in the old plugin (all styles were in one sheet and we tried to avoid inlining). I think it helps future devs know where to look first for style changes/extensions, but I will follow your lead on preference there. Both implementations are just as easy.